Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.h

```diff
@@ -155,6 +157,8 @@ class ModelProcessModelPlayerProxy final
     bool m_autoplay { false };
     bool m_loop { false };
     double m_playbackRate { 1.0 };
+
+    RefPtr<WebCore::SharedBuffer> m_environmentMapData;
```

Comment:
Since `m_environmentMapData` is something that you reset after you've passed it over to `WKSRKEntity` but not represent the current state of the environment map, I wonder if it should be named to `m_pendingEnvironmentMapData` to be clearer.

Reply:
Changed.

Reply:
Looks like the `m_environmentMapData` in `HTMLModelElement` got renamed instead of this one in `ModelProcessModelPlayerProxy`. We can just fix this in a later patch in this area.